.github/dependabot.yml

```diff
@@ -49,6 +49,7 @@ updates:
          patterns:
            - "@jest/*"
            - "@testing-library/*"
+           - "@swc/*"
            - "jest"
            - "jest-*"
            - "ts-jest"
```

Comment:
**thought:** we could remove ts-jest as this is no longer a dependency of the project